### PR TITLE
Update hololens-calibration.md

### PR DESCRIPTION
**Pull request by magdavuko**
- Corrections to the calibration description: not acceptable to close eyes (for longer period) during calibration 
- Adding instructions to disable calibration prompt if calibration is failing. 
- Added info on how many calibrations can be stored locally.

### CODE REVIEW

**Comment by e0i** [comment]
@scooley Can you merge this one if you have the permissions? Thank you in advance, and stay healthy.

File: devices/hololens/hololens-calibration.md [hunk 1]
```diff
@@ -83,11 +83,12 @@ If calibration is unsuccessful try:
 - Moving objects in your visor out of the way (such as hair)
 - Turning on a light in your room or moving out of direct sunlight
 
-If you followed all guidelines and calibration is still failing, please let us know by filing feedback in [Feedback Hub](hololens-feedback.md).
+If you followed all guidelines and calibration is still failing, you can disable the calibration prompt in Settings. Please also let us know by filing feedback in [Feedback Hub](hololens-feedback.md).
 
 ### Calibration data and security
 
 Calibration information is stored locally on the device and is not associated with any account information. There is no record of who has used the device without calibration. This mean new users will get prompted to calibrate visuals when they use the device for the first time, as well as users who opted out of calibration previously or if calibration was unsuccessful.
+The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be automatically deleted.
```
Comment (illfated) [inline devices/hololens/hololens-calibration.md:91]: I am fairly sure the two words at the end of this sentence are normally used in the opposite order:
```suggestion
The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be deleted automatically.
```

**Comment by illfated** [comment]
@magdavuko : Sorry for bothering you with this seemingly cosmetic detail. Would you be able to commit my suggestion, if you find it appropriate?

**Comment by magdavuko** [comment]
Committed.
Magdalena
________________________________

**Comment by illfated** [comment]
#sign-off

File: devices/hololens/hololens-calibration.md [hunk 2]
```diff
@@ -83,11 +83,12 @@ If calibration is unsuccessful try:
 - Moving objects in your visor out of the way (such as hair)
 - Turning on a light in your room or moving out of direct sunlight
 
-If you followed all guidelines and calibration is still failing, please let us know by filing feedback in [Feedback Hub](hololens-feedback.md).
+If you followed all guidelines and calibration is still failing, you can disable the calibration prompt in Settings. Please also let us know by filing feedback in [Feedback Hub](hololens-feedback.md).
 
 ### Calibration data and security
 
 Calibration information is stored locally on the device and is not associated with any account information. There is no record of who has used the device without calibration. This mean new users will get prompted to calibrate visuals when they use the device for the first time, as well as users who opted out of calibration previously or if calibration was unsuccessful.
+The device can store up to 50 calibration profiles locally. After this number has been reached, the oldest unused profile will be deleted automatically.
```
Comment (Teresa-Motiv) [inline devices/hololens/hololens-calibration.md:91]: Please add two spaces (or a CR) to the end of the preceding paragraph. Update the new text as follows:
The device can locally store up to 50 calibration profiles. After this number is reached, the device automatically deletes the oldest unused profile.

Comment (illfated) [reply devices/hololens/hololens-calibration.md:91]: Based on the comment above, this is the requested change:
```suggestion

The device can locally store up to 50 calibration profiles. After this number is reached, the device automatically deletes the oldest unused profile.
```

**Pull request merged**